package-examples/ghost/ghost/secret-example-ghost.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: example-ghost
+  namespace: example
+type: Opaque
+data:
+  ghost-password: amRYdHhYYVFueg==
```

Comment:
We should not have this secret in this package. 

The Good thing is when user will deploy this package, the deployment that needs this secret will be blocked until the secret is available.

So for now secret can be provisioned out-of-band manually. And in later part of our CUJ, we can add a secret manager to manage the secret. WDYT ?

Reply:
This is discussed offline...treating secret as an external resources.